db_seed.sql

```diff
@@ -12,7 +12,7 @@ INSERT INTO server VALUES
     --(3, 'Overleaf', 'https://cdn.overleaf.com/img/ol-brand/overleaf_og_logo.png', 'This is our nice and beautiful Overleaf server.', 'start_cmd', 'stop_cmd', 0),
     --(4, 'Minecraft 2', 'https://theme.zdassets.com/theme_assets/2155033/bc270c23058d513de5124ffea6bf9199af7a2370.png', 'This is our nice and beautiful Minecraft 2 server.', 'start_cmd', 'stop_cmd', 1),
     --(5, 'Ubuntu Server', 'https://cdn-icons-png.flaticon.com/512/5969/5969282.png', 'This is our nice and beautiful Ubuntu server.', 'start_cmd', 'stop_cmd', 0)
-    (1, "Navarcles' Arduino: LED", "https://navlab.ericroy.net/static/media/led.png", "This is the LED that can be found in Isaac's Arduino.", '', '', 0)
+    (1, "Navarcles' Arduino: LED", "https://navlab.ericroy.net/static/media/led.png", "This is the LED that can be found in Isaac's Arduino.", '', 0)
```

Comment:
If endpoing_url is required (see server.py), you MUST put an endpoint url, otherwise put NULL here and don't force people to insert and endponit URL